src/app/home/home.component.html

```diff
@@ -1,45 +1,69 @@
 <div class="container">
   <app-loader [isLoading]="isLoading" size="1.5"></app-loader>
-  <mat-card *ngIf="!isLoading">
-    <mat-card-content>
-      <div class="title">
-        <img src="assets/img/cloud.png" width="12%" alt="Cloudiefy" />
-        <mat-card-subtitle>
-            <p translate>I am Cloudiefy, a web app created to catch the weather. Please search a country:</p>
-          </mat-card-subtitle>
-      </div>
-      <div class="container">
-        <form [formGroup]="weatherForm">
-          <mat-form-field>
-            <input matInput placeholder="Country" formControlName="country">
-            <mat-error *ngIf="country.hasError('required')">
-              Country is <strong>required</strong>
-            </mat-error>
-          </mat-form-field>
-          <button mat-raised-button color="accent" (click)="isLoading = true">Go!</button>
-        </form>
-
-
-
-      
-        <!--mat-form-field>
-          <mat-select placeholder="Select">
-            <mat-option value="option">Option</mat-option>
+  <div *ngIf="!isLoading">
+    <div class="title">
+        <img src="assets/img/cloud.png" width="7%" alt="Cloudiefy" />
+    </div>
+    <div class="container">
+      <form [formGroup]="weatherForm">
+        <mat-form-field>
```

Comment:
I like material design, but since this is an exercise to practice web technologies, what do you think if we remove it and create your own html/css mockup :smile:  ?

Reply:
Sounds like a very good idea! Because I have lot of things to improve in CSS! Thank you Majo!